src/components/CompareResults/ResultsTitle.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+
+import EditIcon from '@mui/icons-material/EditOutlined';
+import { Input } from '@mui/material';
+import { style } from 'typestyle';
+
+import useRawSearchParams from '../../hooks/useRawSearchParams';
+import { FontsRaw, Colors } from '../../styles';
+
+export const ResultsTitle = () => {
+  const [searchParams, updateSearchParams] = useRawSearchParams();
+  const [resultsTitle, setResultsTitle] = useState(
+    searchParams.get('title') || 'Results',
+  );
+  const [isEditing, setIsEditing] = useState(false);
+
+  const styles = {
+    title: style({
+      ...FontsRaw.HeadingXS,
+      fontWeight: 700,
+      letterSpacing: '-0.01em',
+      margin: 0,
```

Comment:
I think they should be applied on the title span only, but not the full title div containign the icon.